### PR TITLE
New version: MSI.Kombustor.4 version 4.1.27

### LINKED ISSUE #137774: [Update Request]: MSI.Kombustor.4
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

MSI.Kombustor.4

### Package Version

4.1.27

### Please describe the changes you would like to see

Download Page: https://www.geeks3d.com/furmark/kombustor/downloads/
Download URL: https://gpuscore.top/msi/MSI_Kombustor4_Setup_v4.1.27.0_x64.exe

### PR DESCRIPTION
**Pull request by Exorcism0666**
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.0.3 :rocket:

- Resolve: #137774

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137801)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: MSI.Kombustor.4
+PackageVersion: 4.1.27
```
Comment (KaranKad) [inline manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.yaml:5]:
```suggestion
PackageVersion: 4.1.27.0
```

File: manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: MSI.Kombustor.4
+PackageVersion: 4.1.27
```
Comment (KaranKad) [inline manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml:5]:
```suggestion
PackageVersion: 4.1.27.0
```
Edit folder name as well

File: manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-08-27
+AppsAndFeaturesEntries:
+- DisplayName: MSI Kombustor 4.1.20.0 (64-bit)
```
Comment (KaranKad) [inline manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml:16]:
```suggestion
- DisplayName: MSI Kombustor 4.1.27.0 (64-bit)
```

File: manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: MSI.Kombustor.4
+PackageVersion: 4.1.27
```
Comment (KaranKad) [inline manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.1.27.0
```

**Review by KaranKad — CHANGES_REQUESTED** [review]

File: manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml (hunk 2 above)
Comment (KaranKad) [reply manifests/m/MSI/Kombustor/4/4.1.27/MSI.Kombustor.4.installer.yaml:5]: @Exorcism0666

**Review by KaranKad — APPROVED** [review]

**Review by KaranKad — CHANGES_REQUESTED** [review]
Folder structure needs to be renamed to 4.1.27.0

**Pull request closed without merging**